evap/student/templates/student_vote.html

```diff
@@ -250,50 +250,44 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
             updateLastSavedLabel();
             setInterval(updateLastSavedLabel, 1000);
 
-            initTextAnswerWarnings(document.querySelectorAll("#student-vote-form textarea"), JSON.parse($("#text-answer-warnings").text()));
+            initTextAnswerWarnings(document.querySelectorAll("#student-vote-form textarea"), JSON.parse(document.getElementById("text-answer-warnings").textContent));
 
-            var form = $('#student-vote-form');
+            var form = document.getElementById('student-vote-form');
+            let submitListener = (event) => {
```

Comment:
Maybe the handler should be `async` so that we don't have the nested `then` below?

Reply:
Live discussion indicated event handlers can't be async functions directly -- still change stuff here? I'm fine with how it currently is.

Reply:
Maybe we will come up with a `replaceWithAsyncHandler` helper in the future, if this occurs more frequently? Something like
```typescript
const replaceWithAsyncHandler = handler => event => {
    event.preventDefault();
    // Run async handler in the background now.
    handler();
};
```